@navikt/ds-tokens/src/index.js

```diff
@@ -36,15 +51,15 @@ module.exports = {
         (colors, [name, color]) => ({
           ...colors,
           [name]: {
-            90: { value: darken(color, 80) },
-            80: { value: darken(color, 60) },
-            70: { value: darken(color, 40) },
-            60: { value: darken(color, 20) },
+            90: { value: darkColor(color, 3) },
```

Comment:
Jeg stemmer for å gå over til samme naming convention som material design og tailwind (50, 100, 200, ..., 800, 900): https://material.io/design/color/the-color-system.html#color-theme-creation https://tailwindcss.com/docs/customizing-colors#overview

Mest fordi fargene i tailwind-implementasjonen min av designsystemet da får navn som passer. Tanker?

Reply:
For våre farger vil det sannsynligvis bli uten 50 først, og så legger vi kanskje til det senere.

Reply:
Tenker det er en god ide, ser de fleste systemer bruker lignende navn for sine farger. Hva tenker du om å gjøre denne endringen i en separat PR der vi går gjennom design-tokene vi har til nå å rydder opp i dem. Unngår da at denne PRen har velding mange breaking changes. 
Tenker da å spesifisere dem mer noe lignende https://spectrum.adobe.com/page/design-tokens/ med global/alias/component tokens.

Reply:
Høres ut som en god plan. 👍